### ISSUE TITLE
[BUG] Script repository broken

### ISSUE THREAD

**Issue by flexid2000** [pbek/QOwnNotes#2770]
### Actual behaviour

The command *search for scripts in repository* does not show the list of scripts to be installed. It does not show anything.

![screenshot](https://i.imgur.com/pgntmu7.jpg)

### Expected behaviour

The command should show the list.

### Logs

Nothing specific:

```
[Apr 12 20:28:37] [debug]: Reply from  "/search/code"
[Apr 12 20:28:37] [debug]: Reply from code search
```

### Remark

Installing a script manually is still possible.  
I don't think it's a duplicate of [2747](https://github.com/pbek/QOwnNotes/issues/2747) since I'm not living in India.

### QOwnNotes versions

- 23.4.1 OBS build 1026 for Debian Testing
- 23.4.2 AUR build 1027 for Arch Linux

**Comment by pbek** [comment]
Still does look like the github api got blocked. 😬

**Comment by pbek** [comment]
Can you please post the output from the debug settings that you can copy when you get into the settings dialog in QOwnNotes and head over to the *Debug* section of it.

**Comment by flexid2000** [comment]
Here you go:
```
[Apr 12 23:02:27] [debug]: Set proxy configuration to use NO proxy
[Apr 12 23:02:27] [debug]: initShortcuts  - 'menuBarIsVisible':  true
[Apr 12 23:02:27] [debug]: cloudConnection:  CloudConnection: <id>1 <name>"Standard" <serverUrl>"" <username>"" <accountId>"" <priority>1
[Apr 12 23:02:28] [debug]: initShortcuts  - 'menuBarIsVisible':  true
[Apr 12 23:02:28] [debug]: updatePanelMenu  - 'updatePanelMenu'
[Apr 12 23:02:28] [scripting]: loading script file: $HOME/.local/share/PBE/QOwnNotes/scripts/pandoc-export/pandoc-export.qml
[Apr 12 23:02:28] [scripting]: loading script file: $HOME/.local/share/PBE/QOwnNotes/scripts/atx-headline/atx-headline.qml
[Apr 12 23:02:28] [scripting]: loading script file: $HOME/.local/share/PBE/QOwnNotes/scripts/export-checkboxes/export-checkboxes.qml
[Apr 12 23:02:28] [scripting]: loading script file: $HOME/.local/share/PBE/QOwnNotes/scripts/favorite-note/favorite-note.qml
[Apr 12 23:02:28] [scripting]: loading script file: $HOME/.local/share/PBE/QOwnNotes/scripts/file-export/file-export.qml
[Apr 12 23:02:28] [scripting]: loading script file: $HOME/.local/share/PBE/QOwnNotes/scripts/fix-list-numbers/fix-list-numbers.qml
[Apr 12 23:02:28] [scripting]: loading script file: $HOME/.local/share/PBE/QOwnNotes/scripts/image-preview/image-preview.qml
[Apr 12 23:02:28] [scripting]: loading script file: $HOME/.local/share/PBE/QOwnNotes/scripts/insert-toc/insert-toc.qml
[Apr 12 23:02:28] [scripting]: loading script file: $HOME/.local/share/PBE/QOwnNotes/scripts/italicize/italicize.qml
[Apr 12 23:02:28] [scripting]: loading script file: $HOME/.local/share/PBE/QOwnNotes/scripts/note-link-to-clipboard/note-link-to-clipboard.qml
[Apr 12 23:02:28] [scripting]: loading script file: $HOME/.local/share/PBE/QOwnNotes/scripts/outline-numbering/outline-numbering.qml
[Apr 12 23:02:28] [scripting]: loading script file: $HOME/.local/share/PBE/QOwnNotes/scripts/select-section/select-section.qml
[Apr 12 23:02:28] [scripting]: loading script file: $HOME/.local/share/PBE/QOwnNotes/scripts/text-highlights/text-highlights.qml
[…]
[Apr 12 23:02:28] [debug]: setupNoteFolderTables  - 'database version':  15
[Apr 12 23:02:28] [debug]: cloudConnection:  CloudConnection: <id>1 <name>"Standard" <serverUrl>"" <username>"" <accountId>"" <priority>1
[Apr 12 23:02:28] [debug]: loadNoteDirectoryList
[Apr 12 23:02:28] [debug]: reloadTagTree
[Apr 12 23:02:28] [debug]: reloadTagTree  - 'noteSubFolderIds':  QVector()
[Apr 12 23:02:28] [debug]: filterNotesByNoteSubFolders  - 'noteSubFolderIds':  QVector(1)
[Apr 12 23:02:28] [debug]: reloadNoteSubFolderTree
[Apr 12 23:02:28] [debug]: reload
[Apr 12 23:02:28] [debug]: filterNotesByNoteSubFolders  - 'noteSubFolderIds':  QVector(1)
[Apr 12 23:02:28] [debug]: loadNoteDirectoryList
[Apr 12 23:02:28] [debug]: reloadTagTree
[Apr 12 23:02:28] [debug]: reloadTagTree  - 'noteSubFolderIds':  QVector(1)
[Apr 12 23:02:28] [debug]: filterNotesByNoteSubFolders  - 'noteSubFolderIds':  QVector(1)
[Apr 12 23:02:28] [debug]: reloadNoteSubFolderTree
[Apr 12 23:02:28] [debug]: reload
[Apr 12 23:02:28] [debug]: filterNotesByNoteSubFolders  - 'noteSubFolderIds':  QVector(1)
[Apr 12 23:02:28] [debug]: reloadNoteSubFolderTree
[Apr 12 23:02:28] [debug]: reload
[Apr 12 23:02:28] [debug]: updatePanelMenu  - 'updatePanelMenu'
[Apr 12 23:02:28] [debug]: updatePanelMenu  - 'updatePanelMenu'
[Apr 12 23:02:28] [debug]: updatePanelMenu  - 'updatePanelMenu'
[Apr 12 23:02:28] [debug]: updatePanelMenu  - 'updatePanelMenu'
[Apr 12 23:02:28] [debug]: updatePanelMenu  - 'updatePanelMenu'
[Apr 12 23:02:28] [info]: "QOwnNotes-Server arbeitet auf Port 22222"
[Apr 12 23:02:28] [debug]: on_noteTreeWidget_itemSelectionChanged
```

**Comment by pbek** [comment]
> Can you please post the output from the debug settings that you can copy when you get into the settings dialog in QOwnNotes and head over to the *Debug* section of it.

This is what I asked for 😅

**Comment by pbek** [comment]
But at least I now know that you already have scripts from the repository installed...

**Comment by pbek** [comment]
Yes, should be fixed since 23.5.0 or so.
